docs/contributing.md

```diff
@@ -8,19 +8,13 @@
 3. Did you document your change? Update the README.md?
 
 ### Initial Setup
-After checking out the repo, making sure you have rvm and nvm setup (setup ruby and node), 
-cd to `spec/dummy` and run `bin/setup` to install dependencies.  
-You can also run `bin/console` for an interactive prompt that will allow you to experiment. 
+After checking out the repo, making sure you have rvm and nvm setup (setup ruby and node), cd to `spec/dummy` and run `bin/setup` to install dependencies. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
 
 ### Starting the Dummy App
-To run the test app, it's **CRITICAL** to not just run `rails s`. You have to run `foreman start`. 
-If you don't do this, then `webpack` will not generate a new bundle, 
-and you will be seriously confused when you change JavaScript and the app does not change. 
+To run the test app, it's **CRITICAL** to not just run `rails s`. You have to run `foreman start`. If you don't do this, then `webpack` will not generate a new bundle, and you will be seriously confused when you change JavaScript and the app does not change.
 
 ### Install and Release
-To install this gem onto your local machine, run `bundle exec rake install`. To release a new version, 
-update the version number in `version.rb`, and then run `bundle exec rake release`, 
-which will create a git tag for the version, push git commits and tags, and push the `.gem` file to [rubygems.org](https://rubygems.org).
+To install this gem onto your local machine, run `bundle exec rake install`. To release a new version, update the version number in `version.rb`, and then run `bundle exec rake release`, which will create a git tag for the version, push git commits and tags, and push the `.gem` file to [rubygems.org](https://rubygems.org).
```

Comment:
Let's move the release part to a separate file in the docs directory. I'm the only one that's going to be doing releases, so this does not apply to anybody else, on the release part.